independent-projects/bootstrap/maven-resolver/src/main/java/io/quarkus/bootstrap/resolver/BootstrapAppModelResolver.java

```diff
@@ -281,6 +291,31 @@ public boolean visitLeave(DependencyNode node) {
                 .build();
     }
 
+    private void collectPlatformProperties(AppModel.Builder appBuilder, List<Dependency> managedDeps)
+            throws AppModelResolverException {
+        for (Dependency d : managedDeps) {
+            // assuming platform properties artifacts have the same reserved groupId
+            Artifact artifact = d.getArtifact();
+            if (artifact.getArtifactId().endsWith(BootstrapConstants.PLATFORM_PROPERTIES_ARTIFACT_ID_SUFFIX)) {
+                final AppArtifact appArtifact = artifact.getExtension().equals("properties") ? toAppArtifact(artifact)
```

Comment:
is the extension here the file extension?

Reply:
yes, it's the file extension.